prawns/governance/src/handlers/democracy.Proposed.event.ts

```diff
@@ -55,7 +55,7 @@ export default (network: SubstrateNetwork) =>
     await ctx.store.save(account);
 
     const proposal = new SubstrateDemocracyProposal({
-      id: `${network}:${blockNumber.toString()}:${ctx.event.indexInBlock}`,
+      id: `${network}:${event.proposalIndex}`,
```

Comment:
Is it unique? If so, you can also update the comment in the 
`prawns/governance/schema.graphql`